Source/TutorialController.swift

```diff
@@ -1,26 +1,26 @@
 import UIKit
 import Pages
-import Hex
 
 public class TutorialController: PagesController {
 
-  private var backLayer = [BackViewModel]()
-  private var animationLayer = [Int: [Animation]]()
-  private var animationIndex = -1
+  private var scene = [Content]()
+  private var slides = [SlideController]()
+  private var animations = [Int: [Animation]]()
```

Comment:
What's the `Int` for? 

Also might be better to add some separation after the `:` => `[Int : [Animation]]()`